test/configurations.jl

```diff
@@ -50,7 +50,16 @@
 
     @testset "Number of electrons" begin
         @test num_electrons(c"[He]") == 2
+        @test num_electrons(rc"[Xe]") == 54
```

Comment:
Maybe you meant `@test num_electrons(rc"[Ne]") == 2` here?

Reply:
Ah, yes, a copy paste mistake.

```suggestion
        @test num_electrons(rc"[He]") == 2
```